modules/elb.py

```diff
@@ -0,0 +1,26 @@
+#! /usr/bin/env python3
+from helper import get_all_instances, get_all_rds, get_all_sg, get_all_elbs
+import crayons
+import boto3
+import config
+
+
+def scan():
+    not_used = []
+    flags = []
+
+    response = get_all_elbs()
+    for ELB in response['LoadBalancerDescriptions']:
+        if len(ELB['Instances']) == 0:
+            not_used.append(ELB['LoadBalancerName'])
+            flags.append(crayons.yellow(" Not used"))
+
+    for elb in not_used:
+        if len(flags) > 0:
```

Comment:
You use one global `flags` array for all ELBs. How should this work when you have more than one ELB ?